Sources/Paywalls/Events/Networking/PaywallEventsRequest.swift

```diff
@@ -16,20 +16,39 @@ import Foundation
 /// The content of a request to the events endpoints.
 struct PaywallEventsRequest {
 
-    var events: [Event]
+    var events: [AnyEncodable]
 
-    init(events: [Event]) {
+    init(events: [AnyEncodable]) {
         self.events = events
     }
 
     @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
     init(events: [PaywallStoredEvent]) {
-        self.init(events: events.map { .init(storedEvent: $0) })
+        self.init(events: events.compactMap { storedEvent in
+            switch storedEvent.feature {
+            case .paywalls:
+                guard let event = PaywallEventRequest.Event(storedEvent: storedEvent) else {
+                    return nil
+                }
+                return AnyEncodable(event)
+        })
```

Comment:
there is going to be a case `.customerCenter` here, that will create a `CustomerCenterEventRequest.Event`

(In another PR)